draft-ietf-httpbis-sfbis.md

```diff
@@ -61,7 +61,7 @@ informative:
   RFC9113:
     display: HTTP/2
   HPACK: RFC7541
-  URI: RFC3986
+  ENCODING: RFC4648
```

Comment:
Using "ENCODING" is a bit misleading. If we don't have a catchy name for the spec, I'd suggest just saying RFC4648.